bin/crunchy-postgres-exporter/start.sh

```diff
@@ -214,9 +214,9 @@ else
     fi
 fi
 
-sed -i "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" /tmp/queries.yml
+sed -i -e "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" -e "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT:-100}/g" /tmp/queries.yml
 
-PG_OPTIONS="--extend.query-path=${QUERY_DIR?}/queries.yml  --web.listen-address=:${POSTGRES_EXPORTER_PORT}"
+PG_OPTIONS="--extend.query-path=${QUERY_DIR?}/queries.yml  --web.listen-address=:${POSTGRES_EXPORTER_PORT} --disable-default-metrics --disable-settings-metrics"
```

Comment:
Why is this being changed?

Reply:
The additional parameters were present on the pgmonitor sysconfig file:
https://github.com/CrunchyData/pgmonitor/blob/853afdfc7d7b59a38e4fd20da7bdc058217be5ef/exporter/postgres/sysconfig.postgres_exporter_pg13#L10

I modified it to re-align with the pgmonitor standalone setup.

Reply:
@ssiahetiong Unfortunately the `--disable-settings-metrics` breaks the existing integration. For example, when testing, I found that the active connection dials no longer worked.

While the Postgres Operator uses a lot of the functionality brought by pgMonitor, it does rely on some of the `postgres_exporter` defaults to accomplish it tasks.

Please rebase and remove 0f118f94bdca73f460c8960e2bb83f04e0398e21 from this patchset. Thanks!

Reply:
done